products/couchbase-server.md

```diff
@@ -22,17 +22,17 @@ releases:
 -   releaseCycle: "7.1"
     eol: false
     latest: "7.1.3"
-    latestReleaseDate: 2023-02-01
```

Comment:
I wondered if the auto should not be disabled for this one. The docker image is regularly updated, even for older releases, so the `latestReleaseDate` is really wrong.

Reply:
@captn3m0 any thought on this ?